source/features/ci-link.tsx

```diff
@@ -35,5 +35,6 @@ void features.add(__filebasename, {
 		pageDetect.isEmptyRepo,
 	],
 	awaitDomReady: false,
+	deduplicate: 'has-rgh',
```

Comment:
Why? This should be the default.

Reply:
Since this feature affects the DOM outside of `#repo-content-pjax-container`, we want it to look at `has-rgh` only. Otherwise when `has-rgh-inner` is removed it will run again, even though the repo header hasn't changed.

Reply:
Then my suggested logic doesn't work. The intent was to create logic that works for every feature, but if as you say features need to specify which deduplicator to use, then the logic is no longer sound.